src/lib/components/ProductForm.svelte

```diff
@@ -27,6 +27,7 @@
 	import type { PojoObject } from '$lib/server/pojo';
 	import type { PaymentMethod } from '$lib/server/payment-methods';
 	import { useI18n } from '$lib/i18n';
+	import { toLower } from 'lodash-es';
```

Comment:
```suggestion
	import { toLower } from 'lodash-es';
```

Let's not import anything from lodash on the frontend

you can use ?.toLowerCase() on a string